Documentation/user-guides/getting-started.md

```diff
@@ -93,152 +65,13 @@ spec:
           containerPort: 8080
 ```
 
-Let's expose the application with a Service object which selects all the Pods
+Let's expose the application with a Service object that selects all the Pods
```

Comment:
this paragraph should be moved to the ServiceMonitor section?

Reply:
What about moving the "Using ServiceMonitors" section above PodMonitor? Then we can explain the need for the Service object in this para and move on to that section.

Reply:
works for me.